### ISSUE TITLE
Incompatible release 0.2.7 for Debian 10/11

### ISSUE THREAD

**Issue by nlimpert** [colcon/colcon-defaults#36]
Version tag `0.2.7` released for Debian 10 / 11 doesn't seem to be installable due to `zstd` missing in the respective `dpkg` versions (`1.19.8` in Debian 10, `1.20.12` in Debian 11).

The installation fails as follows:

```
dpkg-deb: error: archive '/tmp/apt-dpkg-install-Hy5Zg5/1-python3-colcon-defaults_0.2.7-1_all.deb' uses unknown compression for member 'control.tar.zst', giving up
```

According to `packages.ros.org` the package has been released in this state as follows:
 - [ROS / Debian 10: `0.2.7-1`](http://packages.ros.org/ros/ubuntu/dists/buster/main/binary-amd64/Packages)
 - [ROS 2 / Debian 10 `0.2.7-1`](http://packages.ros.org/ros2/ubuntu/dists/buster/main/binary-amd64/Packages)

However, [ROS 2 / Debian 11 `0.2.5-1`](http://packages.ros.org/ros2/ubuntu/dists/bullseye/main/binary-amd64/Packages) installs just fine.

Please let me know if there is a more appropriate place to discuss this.

**Comment by nlimpert** [comment]
A similar issue applies to [`colcon-notification`](https://github.com/colcon/colcon-notification).

Version `0.2.15` is listed for Debian 10/11 but only version `0.2.13` installs respectively.

Would you like me to create an additional issue in the respective repository?

**Comment by nlimpert** [comment]
It looks like this is fixed with the release of `0.2.8` :+1:. Thanks!

The issue with `colcon-notification` remains though so I'll submit a similar issue there.
